src/uct/sm/mm/mm_iface.h

```diff
@@ -142,18 +144,18 @@ uct_mm_iface_invoke_am(uct_mm_iface_t *iface, uint8_t am_id, void *data,
  * memory.
  *
  * @param [in] mem_region  pointer to the beginning of the allocated memory.
- * @param [out] fifo_ctl an aligned pointer to the beginning of the ctl struct in the FIFO
  * @param [out] fifo_elems an aligned pointer to the first FIFO element.
  */
-static inline void uct_mm_set_fifo_ptrs(void *mem_region, uct_mm_fifo_ctl_t **fifo_ctl,
-		                                void **fifo_elems)
+static inline void uct_mm_set_fifo_elems_ptr(void *mem_region, void **fifo_elems)
 {
+   uct_mm_fifo_ctl_t *fifo_ctl;
+
    /* initiate the the uct_mm_fifo_ctl struct, holding the head and the tail */
-   *fifo_ctl   = (uct_mm_fifo_ctl_t*) ucs_align_up_pow2
+   fifo_ctl   = (uct_mm_fifo_ctl_t*) ucs_align_up_pow2
```

Comment:
pls extract this to function since it's used multiple times:
`uct_mm_fifo_ctl_t* uct_mm_fifo_ctl(void *mem_region)`